src/docs/uxd/toolbarsFilters/DOC.md

```diff
@@ -1,5 +1,102 @@
+Pre-May 5 (and the delivery of the attribute-value filtering), use faceted filtering.
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/attributeValueFiltering.png?raw=true "attributeValueFiltering")
+*Attribute-value filtering*
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/facetedFiltering.png?raw=true "facetedFiltering")
+*Faceted filtering*
+
+To show the results of the filtering, use [chip groups](https://v2.patternfly.org/documentation/react/components/chipgroup). The chip groups are valid for both attribute-value filtering and faceted filtering.
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/chipGroupsFacetedFilter.png?raw=true "chipGroupsFacetedFilter")
+*Chip groups with a faceted filter*
+
+For a little more detail:
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/filterChips.png?raw=true "filterChips")
+*Mock from Advisor showing filter chips (no badges)*
+
+OTHER DESIGN OPTIONS:
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/labeledFilterState.png?raw=true "labeledFilterState")
+*Another option for filtering states- Using color & a small label change to indicate filtered*
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/filterChipsBadges.png?raw=true "filterChipsBadges")
+*Mock from Advisor showing filter chips (with badges)*
+
+________
+Discussion is archived below
+
+## Toolbars in MCOM ~~(and beyond?)~~
+
+### The Problem
+
+Since there have been different people doing different stuff at different times, the toolbars are a bit all over the place at the moment. It'd be good to decide on a recommended order for all of the stuff that goes in the toolbar so that we can be consistent. PF 3 had some general [recommendations](https://www.patternfly.org/pattern-library/forms-and-controls/toolbar/#design) but I think we can be a little more specific for MCOM.
+
+### What's in the toolbar?
+
+Note: Not all MCOM UIs will have all of these widgets.
+
+- Filter text box
+- Filter dropdown (checkbox filter)
+- "Remediate with Ansible" action
+- Number of items in the table
+- Pagination -- will this only be at the bottom of tables?
+- Export table selector (JSON, CSV)
+- "Hide things that don't affect my systems" checkbox
+
+### Questions
+
+- What order do we want to suggest for the different widgets?
+- Where should pagination go?
+- Do filter chips go below the toolbar?
+- What do we do about bulk actions?
+- How does responsiveness work?
+
+### Examples from our wires
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/configurationAssessment.png?raw=true "configurationAssessment")
```

Comment:
When you delete this section, make sure you ALSO remove the coresponding img files so we don't bloat the app